smoketests/test.Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM postgis/postgis:11-2.5
+
+RUN apt-get update \
+ # install newer packages from backports
+ && apt-get install -y --no-install-recommends \
+    g++ \
+    gcc \
+    libc6-dev \
+    make \
+    pkg-config \
+    libgeos-dev \
+    libleveldb-dev \
+    libprotobuf-dev \
+    osmctools \
+    postgis \
+    osmosis \
+ # install postgresql client
+ && apt-get install -y --no-install-recommends \
+      postgresql-client-$PG_MAJOR \
+ && ln -s /usr/lib/libgeos_c.so /usr/lib/libgeos.so \
```

Comment:
Why do you need this ?

Reply:
imposm needs this library to execute AFAICT